system_tests/pubsub.py

```diff
@@ -135,7 +135,6 @@ def test_create_subscription_w_ack_deadline(self):
     def test_list_subscriptions(self):
         TOPIC_NAME = 'list-sub' + unique_resource_id('-')
         topic = Config.CLIENT.topic(TOPIC_NAME)
-        self.assertFalse(retry_unavailable(topic.exists)())
```

Comment:
That `topic.exists()` call was failing repeatedly on my local machine while testing the `expectedFailure` change.  I don't believe the assertion adds any value (the topic name is unique to each run of this test).